src/api/operator/utils.ts

```diff
@@ -0,0 +1,54 @@
+// Util functions that only pertain to/deal with operator API related stuff
+
+import BigNumber from 'bignumber.js'
+
+import { ONE_BIG_NUMBER } from 'const'
+
+import { RawOrder } from './types'
+
+export type OrderStatus = 'open' | 'filled' | 'expired' | 'partially filled'
+
+// TODO: move to const or app settings?
+const FILLED_ORDER_EPSILON = '0.0001' // == 0.01%
```

Comment:
Yes, in the constants. I wouldn't put it in the settings. 

Actually, the settings is sth I'm not sure we have sorted out since this is multi-app thing. We don't necesarilly have the same setting file as Mesa. We can keep it simple for now